content/en/docs/concepts/scheduling-eviction/assign-pod-node.md

```diff
@@ -492,6 +498,24 @@ overall utilization.
 Read [Pod topology spread constraints](/docs/concepts/scheduling-eviction/topology-spread-constraints/)
 to learn more about how these work.
 
+## Operators
+
+The following are all the logical operators that you can use with the `operator` field for `nodeAffinity` and `podAffinity` as mentioned above.
+
+|    <div style="width:150px">Operator</div>    |    Behaviour    |
```

Comment:
Don't use HTML here. If we need to fix things (we do), that should be a separate mechanism that we define across multiple pages. For example, we might alter our custom `table` shortcode: https://kubernetes.io/docs/contribute/style/hugo-shortcodes/#table-captions